saleor/graphql/invoice/tests/test_invoice_utils.py

```diff
@@ -23,3 +24,33 @@ def test_generate_invoice_number_old_invoice(datetime_mock, order):
 def test_generate_invoice_number_no_existing_invoice(datetime_mock, order):
     datetime_mock.now.return_value = datetime(2020, 7, 23, 12, 59, 59)
     assert generate_invoice_number() == "1/07/2020"
+
+
+class MockInvoicePluginActive:
+    active = True
+
+    @staticmethod
+    def is_event_active(_):
```

Comment:
Why we have `_` in parameters?

Reply:
method `is_event_active` takes one positional argument and will be called with one by `is_event_active_for_any_plugin`, in case of this mock, the parameter is not used inside the method so `_` is used instead of 'event'.